Documentation/cmdref/cilium-agent.md

```diff
@@ -237,11 +237,11 @@ cilium-agent [flags]
       --log-driver strings                                      Logging endpoints to use for example syslog
       --log-opt map                                             Log driver options for cilium-agent, configmap example for syslog driver: {"syslog.level":"info","syslog.facility":"local5","syslog.tag":"cilium-agent"}
       --log-system-load                                         Enable periodic logging of system load
-      --mesh-auth-mtls-listener-port int                        Port on which the Cilium Agent will perfom mTLS handshakes between other Agents
       --mesh-auth-queue-size int                                Queue size for the auth manager (default 1024)
       --mesh-auth-rotated-identities-queue-size int             The size of the queue for signaling rotated identities. (default 1024)
       --mesh-auth-spiffe-trust-domain string                    The trust domain for the SPIFFE identity. (default "spiffe.cilium")
       --mesh-auth-spire-admin-socket string                     The path for the SPIRE admin agent Unix socket.
+      --mesh-mutual-auth-listener-port int                      Port on which the Cilium Agent will perform mutual authentication handshakes between other Agents
```

Comment:
this rename kind of breaks the unwritten rule that all config values are starting with `mesh-auth-*` :thinking: 

do we want to rename it `mesh-auth-mutual-listener-port`? or `mesh-auth-handshake-listener-port`?

or do we want to introduce `mutual` to other config properties.

having a look at https://github.com/cilium/cilium/pull/25761/files#diff-83e9b8b2ccacddb2fbdb28fbcfa148473d05056c0b4c955fc971709b469d7331 kind of unveils the differences in the naming handling between agent config properties & helm values.

Reply:
I think I'm +1 on `mesh-auth-mutual-*` for this type of flag.